sdk/core/Azure.Core/src/Shared/SequentialDelayStrategy.cs

```diff
@@ -1,19 +1,12 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 
 #nullable enable
 
 namespace Azure.Core
 {
-    /// <summary>
-    /// Polling interval is based on the specified sequence.
-    /// Defaults to {1s, 1s, 1s, 2s, 4s, 8s, 16s, 32s}.
-    /// </summary>
-    /// <remarks>Polling interval always follows the given sequence.</remarks>
```

Comment:
Any reason to remove useful documentation? As this is shared source, we SDK devs have to call this. Frankly, I've been a bit annoyed how little internal source is documented. It means I have to pour into source if it's not clear what a member does. It's not helpful to be not helpful.

Reply:
This documentation is no longer accurate as there is no sequence that you can pass. It is hard-coded to the sequence in the code, so I figured there was no point in duplicating it in two places.

Reply:
Also, this is used purely by code-gen. Library authors never need to use this.

Reply:
I went ahead and added some ref docs for the internal strategies.